docs/data/joy/getting-started/tutorial/LoginFinal.js

```diff
@@ -40,58 +40,60 @@ function ModeToggle() {
   );
 }
 
-export default function LoginFinal() {
+export default function LoginFinal(props) {
   return (
     <main>
-      <ModeToggle />
-      <CssBaseline />
-      <Sheet
-        sx={{
-          width: 300,
-          mx: 'auto', // margin left & right
-          my: 4, // margin top & bottom
-          py: 3, // padding top & bottom
-          px: 2, // padding left & right
-          display: 'flex',
-          flexDirection: 'column',
-          gap: 2,
-          borderRadius: 'sm',
-          boxShadow: 'md',
-        }}
-        variant="outlined"
-      >
-        <div>
-          <Typography level="h4" component="h1">
-            <b>Welcome!</b>
-          </Typography>
-          <Typography level="body-sm">Sign in to continue.</Typography>
-        </div>
-        <FormControl>
-          <FormLabel>Email</FormLabel>
-          <Input
-            // html input attribute
-            name="email"
-            type="email"
-            placeholder="johndoe@email.com"
-          />
-        </FormControl>
-        <FormControl>
-          <FormLabel>Password</FormLabel>
-          <Input
-            // html input attribute
-            name="password"
-            type="password"
-            placeholder="password"
-          />
-        </FormControl>
-        <Button sx={{ mt: 1 /* margin top */ }}>Log in</Button>
-        <Typography
-          endDecorator={<Link href="/sign-up">Sign up</Link>}
-          sx={{ fontSize: 'sm', alignSelf: 'center' }}
+      <CssVarsProvider {...props}>
```

Comment:
This demo is an iframe with mode toggle, so need to add dedicated CssVarsProvider (ThemeProvider) to follow the new demo guide.